wordpress-mobile/WordPress-iOS
# Update Automattic-Tracks-iOS to version 0.4.4-beta.2

Hack Week - Pods migration to Swift 5
Ref https://github.com/Automattic/Automattic-Tracks-iOS/pull/119

#### NOTES: 
- PR Is WIP
- will not be merged until the new pod version (`Automattic-Tracks-iOS version 0.4.4-beta.2`) gets pushed.
- Before merging, the `Podfile` will be updated to reference the actual version of `Automattic-Tracks-iOS` instead of a specific branch.

To test:

- Build/run
- verify that tracks are triggered by specific actions

PR submission checklist:
